lib/rubocop/cop/lint/number_conversion.rb

```diff
@@ -49,6 +51,10 @@ def on_send(node)
 
         private
 
+        def allowed_receivers
+          ['Time.now']
```

Comment:
Does the same thing happen with `Time.current` and other similar time methods?

This would be better off as a constant.

Reply:
Ok now that I think about it properly (rather than rushing onto a solution) - this happens for any function that returns a `Time` object - which is almost all methods [here](https://ruby-doc.org/core-2.6/Time.html) plus the ones added by ActiveSupport.

Maybe we can check for receiver's parent being Time? Fortunately `Date` and `DateTime` don't support `to_i`, `to_f` etc.